vuu/src/main/scala/org/finos/vuu/core/VuuServer.scala

```diff
@@ -106,6 +106,16 @@ class VuuServer(config: VuuServerConfig)(implicit lifecycle: LifecycleContainer,
     tableContainer.createAutoSubscribeTable(tableDef)
   }
 
+  def createBasketTable(tableDef: TableDef): DataTable = {
```

Comment:
Hi Emily,

We don't need these explicit calls on the Vuu server, the createTable call turns a table def into a map backed real table. You can use the existing ones for your new tables.

Reply:
yes thanks! i have changed it